src/Shared.Core/Utilities/Extensions.cs

```diff
@@ -277,6 +277,127 @@ public static bool SetFieldValue(this object self, string name, object value)
             {
                 KoikatuAPI.Logger.LogError("Unexpected crash when running events, some events might have been skipped! Reason: " + e);
             }
+        }
+
+        /// <summary>
+        /// Compares two byte arrays for equality in a high-performance manner using unsafe code.
+        /// </summary>
+        /// <param name="a">The first byte array to compare.</param>
+        /// <param name="b">The second byte array to compare.</param>
+        /// <returns>True if the byte arrays are equal, false otherwise.</returns>
+        static public bool SequenceEqualFast(this byte[] a, byte[] b)
```

Comment:
```suggestion
        public static bool SequenceEqualFast(this byte[] a, byte[] b)
```

Reply:
sorry

Reply:
It's not a big deal. You should be able to make VisualStudio automatically detect code style and warn about style issues like this (I use resharper extension for this, but I'm fairly certain basic VS has this feature too in latest versions).

Reply:
resharper is expensive and VisualStudio's formatter has no entry to define the static/public order!

Reply:
You can get resharper for free actually, you just have to show them that you contribute to open source projects. Some contributors to the BepInEx repository did that.